pages/report-repair/[route].js

```diff
@@ -60,6 +80,16 @@ function ReportRepair() {
   const [formError, setFormError] = useState();
   const [requestId, setRequestId] = useState();
 
+
+  const { repairTriageData, isLoading, isError } = useRepairTriageData()
+
+  if (shouldRequestTriageData) {
+    if (isLoading) return <Loader/>
+    if (isError) return <Error
```

Comment:
When an error has occurred, this early return doesn't present a back link and could leave the user confused.

Please ensure there is a back link on the page even when an error has occurred.

Reply:
This is a really good spot! I'll look at this now.

Reply:
This is sorted now.